templates/static-fileserver/metadata/spin-template.toml

```diff
@@ -0,0 +1,15 @@
+manifest_version = "1"
+id = "static-fileserver"
+description = "Serves static files from an asset directory"
+trigger_type = "http"
+
+[new_application]
+supported = false
+
+[add_component]
+[add_component.snippets]
+component = "component.txt"
+
+[parameters]
+http-path = { type = "string", prompt = "HTTP path", default = "/static/...", pattern = "^/\\S*$" }
+asset-path = { type = "string", prompt = "Directory containing static assets to serve", default = "assets", pattern = "^\\S+$" }
```

Comment:
I propose to change this to "files-path" with a prompt "Directory containing the files to serve."

Reply:
Done - thanks!